featuretools/primitives/primitive_base.py

```diff
@@ -101,6 +101,11 @@ def variable_type(self):
             feature = feature.base_features[0]
             return_type = feature.return_type
 
+            # only the original time index should exist
+            # so make this feature's return type just a Datetime
+            if return_type == DatetimeTimeIndex:
```

Comment:
We'd encounter a similar issue with NumericTimeIndex, I think

Reply:
you're right. added